week2/week2-HW/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="kor" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>2주차 과제 _ 투두리스트트</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
+    <link rel="stylesheet" href="./styles/reset.css" />
+  </head>
+  <body>
+    <div class="modal">
+      <div class="modal-contents">
+        <div class="title">이미 완료된 할 일입니다</div>
+        <img
+          src="https://static.vecteezy.com/system/resources/previews/021/494/711/non_2x/cute-smile-clover-with-red-cheek-free-png.png"
+        />
+        <button class="modal-btn">확인</button>
+      </div>
+    </div>
+    <header>재림's todolist</header>
```

Comment:
```suggestion
    <header><h1>재림's todolist</h1></header>
```

h1 태그로 텍스트를 감싸주는 건 어떨까요? 해당 텍스트가 제목의 의미를 갖고 있다고 생각해요!